.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   build_lint_and_test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
do you think it's better to downgrade or upgrade here?  if we upgrade, after oct 31 we should be able to stay with `ubuntu-latest`

Reply:
ubuntu-24.04 seems to fail, which is strange because it should have been our latest at least before now.

Reply:
Updated. Thank you.